src/model/image-tag.test.ts

```diff
@@ -64,13 +70,27 @@ describe('ImageTag', () => {
     it('returns the specific build platform', () => {
       const image = new ImageTag({ version: '2019.2.11f1', platform: 'WebGL' });
 
-      expect(image.toString()).toStrictEqual(`${defaults.image}:2019.2.11f1-webgl-0`);
+      switch (process.platform) {
+        case 'win32':
+          expect(image.toString()).toStrictEqual(`${defaults.image}:windows-2019.2.11f1-webgl-0`);
+          break;
+        case 'linux':
+          expect(image.toString()).toStrictEqual(`${defaults.image}:2019.2.11f1-webgl-0`);
+          break;
```

Comment:
Just a note to convey context:

This is where it becomes tricky. Over time each target platform may dictate which base platform works best for it. Perhaps we can use some magic by then to reflect this in user's workflows.

This means that over time we'll have to become very specific about which target platform uses which base platform. At that point this would likely become `${defaults.image}:ubuntu-2019.2.11f1-webgl-0`, with the `ubuntu` part to be specific.

For now we'll just assume that the default is always ubuntu. Just starting the thought engine on how to architect it over time.